test/parallel/test-require-deps-deprecation.js

```diff
@@ -39,6 +42,15 @@ for (const m of deprecatedModules) {
   } catch (err) {}
 }
 
+// Instead of checking require, check that resolve isn't pointing toward
+// /node/deps, as user might already have node installed with acorn in
+// require.resolve range
 for (const m of deps) {
-  assert.throws(() => { require(m); }, /^Error: Cannot find module/);
+  let path = '';
+  try {
+    path = require.resolve(m);
+  } catch ($e) {
+    assert.ok($e);
```

Comment:
nit: I don't think we ever use the `$e` convention. `err` is fine.

Reply:
Damn, sorry, too much PHP lately I think...